contracts/rust/Cargo.toml

```diff
@@ -63,7 +63,7 @@ rand = "0.8.4"
 rand_chacha = "0.3.1"
 reef = { git = "https://github.com/EspressoSystems/reef.git", tag = "0.2.2" }
 regex = "1.5.5"
-seahorse = { git = "https://github.com/EspressoSystems/seahorse.git", tag = "0.2.5" }
+seahorse = { git = "https://github.com/EspressoSystems/seahorse.git", branch = "mathis-atomicstore-0.1.3" }
```

Comment:
Should this branch maybe be tagged instead?

Reply:
I'm waiting for the seahorse PR to be merged, will update afterwards.